.github/workflows/system-tests-against-emulator.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License..
+
```

Comment:
If `set -eo pipefail` does not work, how about just set `set -e`? 

```bash
# Fail on any error
set -eo pipefail

# Display commands being run
set -x
```

Reply:
I thought it wouldn't let me know any `set` commands but this worked so I made the change.